meson.build

```diff
@@ -0,0 +1,87 @@
+project(
+  'fvwm3',
+  'c',
+
+  version: '1.0.5',
+  license: 'GPLv2',
+  meson_version: '>=0.60.0',
+  default_options: [
+    'c_std=c11',
+    'warning_level=2',
+    'werror=false',
+    ],
+  )
+
+add_project_arguments(
+  [
+    '-Wno-unused-parameter',
+    '-Wno-missing-braces',
+    '-Wno-unused-result',
+    '-Wvla',
+    '-Wundef',
+    ],
+  language: 'c',
+  )
+
+add_project_arguments('-D_GNU_SOURCE', language : 'c')
+add_project_arguments('-D_POSIX_C_SOURCE=200809L', language: 'c')
+
+cc = meson.get_compiler('c')
+
+datadir = get_option('datadir')
+sysconfdir = get_option('sysconfdir')
+prefix = get_option('prefix')
+
+version = '"@0@"'.format(meson.project_version())
+git = find_program('git', native: true, required: false)
+if git.found()
+  git_commit = run_command([git, 'rev-parse', '--short', 'HEAD'], check: false)
+        git_branch = run_command([git, 'rev-parse', '--abbrev-ref', 'HEAD'], check: false)
+        if git_commit.returncode() == 0 and git_branch.returncode() == 0
+          version = '"@0@-@1@ (" __DATE__ ", branch \'@2@\')"'.format(
+            meson.project_version(),
+            git_commit.stdout().strip(),
+            git_branch.stdout().strip(),
+            )
+        endif
+endif
+add_project_arguments('-DVERSIONINFO=@0@'.format(version), language: 'c')
+
+# Dependencies - REQUIRED
+libevent = dependency('libevent', required: true)
+libx11 = dependency('x11', required: true)
+libxt = dependency('xt', required: true)
+xft = dependency('xft', required: true)
+xrandr = dependency('xrandr', required: true)
+xrender = dependency('xrender', required: true)
```

Comment:
Note that the default value of `required` is true, so you don't need to spell it out.

Reply:
Hi @eli-schwartz 

Thanks!  Fixed in the next push.